src/structures/StickerPack.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
+
+const Base = require('./Base');
+const Sticker = require('./Sticker');
+const Collection = require('../util/Collection');
+const SnowflakeUtil = require('../util/SnowflakeUtil');
+
+/**
+ * Represents a pack of standard stickers.
+ * @extends {Base}
+ */
+class StickerPack extends Base {
+  /**
+   * @param {Client} client The instantiating client
+   * @param {Object} pack The data for the sticker pack
+   */
+  constructor(client, pack) {
+    super(client);
+    /**
+     * The ID of the sticker pack
+     * @type {Snowflake}
+     */
+    this.id = pack.id;
+
+    /**
+     * The stickers in the pack
+     * @type {Collection<Snowflake, Sticker>}
+     */
+    this.stickers = new Collection(pack.stickers.map(s => [s.id, new Sticker(client, s)]));
+
+    /**
+     * The name of the sticker pack
+     * @type {string}
+     */
+    this.name = pack.name;
+
+    /**
+     * The ID of the pack's SKU
+     * @type {Snowflake}
+     */
+    this.skuID = pack.sku_id;
+
+    /**
+     * The ID of a sticker in the pack which is shown as the pack's icon
+     * @type {Snowflake}
+     */
+    this.coverStickerID = pack.cover_sticker_id;
+
+    /**
+     * The description of the sticker pack
+     * @type {string}
+     */
+    this.description = pack.description;
+
+    /**
+     * The ID of the sticker pack's banner image
+     * @type {?Snowflake}
+     */
+    this.bannerID = pack.banner_asset_id;
```

Comment:
Is this property nullable? Can it be undefined?

Reply:
afaik this is always present and non-null, will ask for confirmation

Reply:
ok got confirmation that this is not optional or nullable, so resolving these

Reply:
If this is not nullable why the type is `?Snowflake`